STABLE_RELEASES.md

```diff
@@ -0,0 +1,140 @@
+# Stable Releases
+
+*Stable Release Series* continue to receive bug fixes until they reach **End Of Life**.
+
+Only the following release series are currently supported and receive bug fixes:
+
+* **1.0** is the latest stable release.
+
+The **1.0** release series is maintained in compliance with the **Stable Release Policy** as described in this document.
+
+## Stable Release Policy
+
+This policy presently applies *only* to the following release series:
+
+* **1.0**
+
+### Point Releases
+
+Once a Desmos release has been completed and published, updates for it are released under certain circumstances
+and must follow the [Point Release Procedure](CONTRIBUTING.md).
+
+### Rationale
+
+Unlike in-development `master` branch snapshots, **Desmos** releases are subject to much wider adoption,
+and by a significantly different demographic of users. During development, changes in the `master` branch
+affect Desmos users, application developers, early adopters, and other advanced users that elect to use
+unstable experimental software at their own risk.
+
+Conversely, users of a stable release expect a high degree of stability. They build their applications on it, and the
+problems they experience with it could be potentially highly disruptive to their projects.
+
+Stable release updates are recommended to the vast majority of developers, and so it is crucial to treat them
+with great caution. Hence, when updates are proposed, they must be accompanied by a strong rationale and present
+a low risk of regressions, i.e. even one-line changes could cause unexpected regressions due to side effects or
+poorly tested code. We never assume that any change, no matter how little or non-intrusive, is completely exempt
+of regression risks.
+
+Therefore, the requirements for stable changes are different from those that are candidates to be merged in
```

Comment:
maybe this should be edited considering we're not developing an SDK?

Reply:
I think it's worth keeping them since we should grant a high grade of stability due to the fact that Desmos will be used as an SDK by different other projects as well (ie. all the DApps that will be developed on top of it)